### PR TITLE
Added wrong-import-order message example

### PR DESCRIPTION
**Pull request by matusvalo**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|     | :bug: Bug fix          |
|     | :sparkles: New feature |
|     | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Refs #5953

### CODE REVIEW

**Comment by matusvalo** [comment]
isort pre-commit hook "fixed" bad.py file causing it is not showing bad case anymore :-(

**Comment by matusvalo** [comment]
Do we want to exclude isort per file

https://github.com/PyCQA/pylint/blob/0afa5bc5ea8641cce751b8f995fe8ce682f397fa/.pre-commit-config.yaml#L40

or we will exclude whole  `doc/data/messages` directory?

**Comment by Pierre-Sassoulas** [comment]
> we will exclude whole doc/data/messages directory?

We could exclude all the ``bad.py`` first is the 'exclude' become hard to manage

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
-        exclude: doc/data/messages/r/reimported/bad.py
+        exclude: doc/data/messages/r/reimported/bad.py|doc/data/messages/w/wrong-import-order/bad.py
```
Comment (Pierre-Sassoulas) [inline .pre-commit-config.yaml:40]:
```suggestion
        exclude: doc/data/messages/(r/reimported|w/wrong-import-order)/bad.py
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Not sure about the regex, I'll revert if it breaks the existing code :)

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you @matusvalo !

**Pull request merged**